packages/nc-gui/composables/useViews.ts

```diff
@@ -2,7 +2,7 @@ import type { TableType, ViewType } from 'nocodb-sdk'
 import type { MaybeRef } from '@vueuse/core'
 import { ref, unref, useNuxtApp, watch } from '#imports'
 
-export function useViews(meta: MaybeRef<TableType | undefined>) {
+function _useViews(meta: MaybeRef<TableType | undefined>) {
```

Comment:
why there is such change?

Reply:
removed - was a left-over from a former approach and discussion where there plan was to move the right side bar also to the left when in mobile mode and afai remember we had to share the same useView composable/state across siblings in the component tree then.